ci/pre-behat.sh

```diff
@@ -0,0 +1,28 @@
+WORDPRESS_SITE_DIR=${WORDPRESS_SITE_DIR-/tmp/wordpress}
+
+# Used when waiting for stuff
+NAP_LENGTH=1
+SELENIUM_PORT=4444
+
+# Wait for a specific port to respond to connections.
+wait_for_port() {
+    local PORT=$1
+    while echo | telnet localhost $PORT 2>&1 | grep -qe 'Connection refused'; do
+        echo "Connection refused on port $PORT. Waiting $NAP_LENGTH seconds..."
+        sleep $NAP_LENGTH
+    done
+}
+
+rm -f /tmp/.X0-lock
+
+Xvfb & export DISPLAY=localhost:0.0
```

Comment:
No, but since we are running selenium in a headless environment we need Xvfb which acts as a virtual display. This is required even if we're not taking screenshots.

On which point, I think Travis (and on their free offering) allow you to upload artefacts to Amazon S3. No reason why this couldn't be used to upload screenshots of test failures.